### PR TITLE
Void

### PR DESCRIPTION
**Pull request by abebeos**

### CODE REVIEW

**Comment by yilunzhang** [comment]
Hmm I thought the name suggests it... I assumed `nknorg/nkn-sdk-cpp` is official and `someone-else/nkn-sdk-cpp` is not official one, not just for nkn but for also most organization repo...

**Comment by abebeos** [comment]
This **someone else** is a **member** of nknorg, and sometimes the organizations members do strange things (e.g. evolving the code under their own personal account/repo, without pushing to original/official repo, see e.g. nMobile).

Confusion is avoided if standard-procedure is followed:

* open source, public projects, public development, always under the organization

File: README.md [hunk 1]
```diff
@@ -1,2 +1,10 @@
 # nkn-sdk-cpp
 NKN SDK for C/C++
+
+## IMPORTANT !!!
+
+This is not the official actual versio.
```
Comment (yilunzhang) [inline README.md:6]: typo: `versio`

**Comment by yilunzhang** [comment]
Oh well, fair point

**Pull request closed without merging**